crawler/file_processing.py

```diff
@@ -1043,22 +1157,22 @@ def row_positive_result_matches_channel_results(self, row, line_number) -> bool:
         # look for positive channel results
         if row.get(FIELD_CH1_RESULT):
             ch_results_present += 1
-            if row.get(FIELD_CH1_RESULT) == 'Positive':
+            if row.get(FIELD_CH1_RESULT) == POSITIVE_RESULT_VALUE:
                 ch_results_positive += 1
 
         if row.get(FIELD_CH2_RESULT):
             ch_results_present += 1
-            if row.get(FIELD_CH2_RESULT) == 'Positive':
+            if row.get(FIELD_CH2_RESULT) == POSITIVE_RESULT_VALUE:
                 ch_results_positive += 1
 
         if row.get(FIELD_CH3_RESULT):
             ch_results_present += 1
-            if row.get(FIELD_CH3_RESULT) == 'Positive':
+            if row.get(FIELD_CH3_RESULT) == POSITIVE_RESULT_VALUE:
                 ch_results_positive += 1
 
         if row.get(FIELD_CH4_RESULT):
             ch_results_present += 1
-            if row.get(FIELD_CH4_RESULT) == 'Positive':
+            if row.get(FIELD_CH4_RESULT) == POSITIVE_RESULT_VALUE:
                 ch_results_positive += 1
 
         # if there are no channel results present in the row we do not need to check further
```

Comment:
Definitely room for some DRYing out here, but I don't want to make you clean up every line that you touched refactoring something else.

Reply:
I think @pjvv is doing some code cleaning in the next branch. Might be worth including this in that clean up?

Reply:
I've done some linting and black auto formatting in branch "maintenance" but will leave refactoring to another branch to keep things sane.